### ISSUE TITLE
bug: notification history window title links to Title highlight group and not to a Snacks highlight group

### ISSUE THREAD

**Issue by freddiehaddad** [folke/snacks.nvim#348]
### Did you check docs and existing issues?

- [X] I have read all the snacks.nvim docs
- [X] I have updated the plugin to the latest version before submitting this issue
- [X] I have searched the existing issues of snacks.nvim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.10.2

### Operating system/version

WIndows 11

### Describe the bug

I'm trying to adjust the notification history popup window title highlight group.  Regardless of which highlight group I change, nothing affects the title **Notification History** except the `Title` highlight group.

### Steps To Reproduce

Change the `Title' highlight group and open the notification history.

### Expected Behavior

Modifying `SnacksNotifierHistoryTitle` highlight group would affect title.

### Repro

```lua
vim.env.LAZY_STDPATH = ".repro"
load(vim.fn.system("curl -s https://raw.githubusercontent.com/folke/lazy.nvim/main/bootstrap.lua"))()

require("lazy.minit").repro({
  spec = {
    { "folke/snacks.nvim", opts = {} },
    -- add any other plugins here
  },
})
```

**Comment by Raagh** [comment]
hi @freddiehaddad I am trying to do the same, can you share how are you accomplishing this change?

**Comment by freddiehaddad** [comment]
Correct. There's actually a lot more highlight groups defined than what the docs mention.  If you view the list of ighlight groups in neovim, you can see them. That's actually how I found the ones I did change.

Back to your question.  I'm not exactly sure what you're asking. As of now, I can't get the Notification History window title colors set how I would like them without modifying the global `Title` highlight group which I don't want to do as that affects other parts of Neovim.

In my bug report, I'm making mention that the title doesn't link to a plugin highlight group.

**Comment by folke** [comment]
It just uses the `FloatTitle` hl group.
Not interested in adding any additonal config for this.
You can change the history styles winhilight yourself if it bothers you
